### PR TITLE
2.3.0: Update dependencies to new hapi scope

### PR DESCRIPTION
**Pull request by drouarb**
Hey,

I updated your hapi plugin since I need it. There is a problem with the tests code coverage but I think it comes from the update of lab.

Coverage

### CODE REVIEW

**Comment by timcosta** [comment]
Hey @drouarb! Thanks for this PR - there are a few issues that would need to be fixed before it can be merged. First is the drop in coverage you mentioned. That should be bumped back up to 100%. Second, this specifies a peerdep of hapi @18.3.1, which is latest. This module is compatible with v17 and v18, which makes this peerdep of "latest" much too strict.

File: package.json [hunk 1]
```diff
@@ -23,17 +23,17 @@
     "node": ">=8.0.0"
   },
   "peerDependencies": {
-    "hapi": ">=17.x.x"
+    "@hapi/hapi": "^18.3.1"
```
Comment (frankthelen) [inline package.json]: Woudn't make it that strict in `peerDependencies`. What about this one:
```
"@hapi/hapi": ">=17.x.x"
```

**Comment by timcosta** [comment]
Works for me. I was planning on getting to this later today if no one else
had completed this PR.

**Comment by timcosta** [comment]
Closing in favor of #33, as this PR hasn't had activity from its owner in some time.

**Pull request closed without merging**